mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -673,13 +706,9 @@ def send_email(self, title, body):
         DeviceApi().send_email(title, body, basename(self.root_dir))
 
     def make_active(self):
-        """Bump skill to active_skill list in intent_service.
-
-        This enables converse method to be called even without skill being
-        used in last 5 minutes.
-        """
-        self.bus.emit(Message('active_skill_request',
-                              {'skill_id': self.skill_id}))
+        if self.skill_control.category == 'undefined':
```

Comment:
add a docstring

Reply:
For what? There is one in skill_control. I am confused.

Reply:
I think he means that the `make_active()` method should have a docstring. In this case it's extra useful since it's one of the methods in the "skill api" which is automatically documented and published on readthedocs: https://mycroft-core.readthedocs.io/en/latest/source/mycroft.html#mycroft.MycroftSkill.make_active

Reply:
Oh I see. Well make_active() is nothing new. It was around before this PR and to be honest I am not sure of its value. This kind of relates to the earlier comment which is how much code do I inherit when I add a PR :-)_

Reply:
No make_active is not a new thing but the PR removes it's docstring...

I guess this is another part of the issue, keeping skill compatibility...There are skills that relies on the functionality of the public methods. Ideally the skill creators that use this method shouldn't be affected even if you extend it's functionality, like you have here.

I think the docstring should remain, but be updated to indicate that this is deprecated method and the new way for keeping skills in converse priority is...

Reply:
Oh, my mistake. I did not mean to remove that method's doc string nor to deprecate it. In fact I believe it still works as always for skills of the 'undefined' category. I was just confused as to what the ask was. Now I understand. I will put back the doc string back in.

Reply:
Doc string added and doc string put back.